geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/converters/FastConverter.scala

```diff
@@ -49,19 +49,7 @@ object FastConverter extends StrictLogging {
     }
 
     val clas = value.getClass
-    var converters = cache.get((clas, binding))
-
-    if (converters == null) {
-      if (clas.eq(binding) || clas == binding || binding.isAssignableFrom(clas)) {
-        converters = Array(IdentityConverter)
-      } else {
-        converters = factories.flatMap(factory => Option(factory.createConverter(clas, binding, null)))
-        if (binding == classOf[String]) {
-          converters = converters :+ ToStringConverter // add toString as a final fallback
-        }
-      }
-      cache.put((clas, binding), converters)
-    }
+    val converters = getConverters(clas, binding)
```

Comment:
What's this change about?

Reply:
i added a method to convert to multiple bindings, picking the first one. this is mainly to support the geotools behavior of trying to bind a date to a 'period' before binding it to a 'date'. the old way would always log warnings that things couldn't be converted to a period, even though that was expected behavior. this way it will only log a warning if it can't convert to any of the allowed bindings